src/import-v1/index.ts

```diff
@@ -548,6 +553,12 @@ export async function importV1TokenOfferingData(db: PgWriteStore) {
 }
 
 export async function handleBnsImport(db: PgWriteStore) {
+  const stacksNodeType = process.env.STACKS_NODE_TYPE;
+  if (stackNodeType === StacksNodeType.Subnet) {
+    logger.error('BNS imports should not be enabled for a Subnet');
```

Comment:
With this behavior, we will log and keep running (as opposed to exiting with error). That seems okay to me, but we should make the error a bit more explicit in that case:
> `BNS imports should not be enabled for a Subnet, skipping ...`